client/my-sites/checkout/checkout/checkout-system-decider.js

```diff
@@ -147,10 +147,7 @@ function shouldShowCompositeCheckout( cart, countryCode, locale, productSlug, is
 
 	// If the URL is adding a product, only allow wpcom plans
 	const slugFragmentsToAllow = [ 'personal', 'premium', 'blogger', 'ecommerce', 'business' ];
```

Comment:
Are we sure these slugs are correct? For example, Premium plan seems to be `value_bundle` (this one is the trickiest). While Business plan is `business-bundle`  and so on.

Reply:
`value_bundle` is the correct actual slug for the product, but `premium` is a special slug that is used by "add plan" urls. These are the only ones it supports, AFAIK. For other products you have to use a renewal link url, which uses actual slugs (but we don't support that yet in composite checkout).

Reply:
Nevermind, looks like we have different slugs thrown around. And these might be valid in some cases.